_includes/configuration/paste_as_text.md

```diff
@@ -0,0 +1,27 @@
+{% if page.title == "Paste plugin" %}
+  {% assign plugin = "paste" %}
+{% else %}
+  {% assign plugin = "powerpaste" %}
+{% endif %}
+### `paste_as_text`
+
+This option enables you to set the default state of the `Paste as text` menu item, which is added by the `{{plugin}}` plugin under the `Edit` menu dropdown. It's disabled by default.
```

Comment:
More pronouns for the paste plugin it seems :cry:

Reply:
:disappointed: